plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -505,10 +505,8 @@ func GetResourceRequest(pod *v1.Pod) *schedulercache.Resource {
 			default:
```

Comment:
feels like this method needs a header comment `// GetResourceRequest calculates maximal individual init containers (since they're serial) and sum of standard pod containers (since theyre running together), and takes the max of the entire set.`